### PR TITLE
Add verified icon

### PR DESCRIPTION
**Pull request by siarie**
Preview
---
2021-10-02-202717_493x426_scrot

### CODE REVIEW

**Comment by ericfennis** [comment]
@siarie Awesome icon! Thanks for this contribution!

File: icons/verified.svg [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M11.997 3a3.599 3.599 0 00-3.044 1.682 3.599 3.599 0 00-.908-.095 3.599 3.599 0 00-2.411 1.051 3.6 3.6 0 00-.947 3.318A3.6 3.6 0 003 12.004a3.6 3.6 0 001.69 3.047 3.6 3.6 0 00.947 3.315 3.599 3.599 0 003.352.96A3.599 3.599 0 0012.002 21a3.599 3.599 0 003.037-1.671 3.599 3.599 0 004.295-4.297A3.6 3.6 0 0021 11.998a3.6 3.6 0 00-1.67-3.035v-.002a3.599 3.599 0 00-4.298-4.292A3.599 3.599 0 0011.997 3z" />
```
Comment (ericfennis) [inline icons/verified.svg:12]: With [SVGOMG](https://jakearchibald.github.io/svgomg/) I was able to minify it a bit.

```suggestion
  <path d="M12 3a3.6 3.6 0 0 0-3.05 1.68 3.6 3.6 0 0 0-.9-.1 3.6 3.6 0 0 0-2.42 1.06 3.6 3.6 0 0 0-.94 3.32A3.6 3.6 0 0 0 3 12a3.6 3.6 0 0 0 1.69 3.05 3.6 3.6 0 0 0 .95 3.32 3.6 3.6 0 0 0 3.35.96A3.6 3.6 0 0 0 12 21a3.6 3.6 0 0 0 3.04-1.67 3.6 3.6 0 0 0 4.3-4.3A3.6 3.6 0 0 0 21 12a3.6 3.6 0 0 0-1.67-3.04v0a3.6 3.6 0 0 0-4.3-4.3A3.6 3.6 0 0 0 12 3z"/>
```

Comment (siarie) [reply icons/verified.svg:12]: Ah yeah. I forgot about that.
Anyway, can you guide me how to use SVGO? I wanted to try it myself, but the result was different from what you suggested.

**Comment by siarie** [comment]
Hmm, :thinking:. Looks like I got something wrong.
I forgot about point 2 in `ICON-DESIGN-GUIDE`

> 2. Icons must have a **1 pixel padding** within the canvas.

But I see many icons have 2 pixel padding (and even 3 pixel). So, which one is correct?
Though, the guide says 1px. But before I fix it, I wanna hear your opinions.

cc: @ericfennis, @locness3

**Comment by ericfennis** [comment]
@siarie I understand your confusion. I think we need to update the text a bit. It should be "Icons must have at least 1 pixel padding within the canvas.". We want a least 1 pixel empty around the canvas to make sure icons will render nicely. Otherwise some icons will look like they are "cut off". 

> There are still icons which don't have the 1 pixel padding. They still need to be refactored

**Review by ericfennis — APPROVED** [review]
So this icon you made is fine!

**Comment by locness3** [comment]
It's 1px minimum.

**Comment by locness3** [comment]
Oh, the name...

**Comment by siarie** [comment]
> Oh, the name...

Hmm?. Did you mean [this PR](https://github.com/lucide-icons/lucide/pull/350)?
The name `piggy-bank-.svg` caused build error. @locness3

**Comment by locness3** [comment]
wth

I meant `verified` is not the kind of name I'd put in lucide. How would you call the outer shape of this icon?

**Comment by siarie** [comment]
Well, I can't think of any other name other than this. 
I saw bootstrap using the name `patch-check` and fontawesome using `badge-check`. Do you have any suggestion?

> How would you call the outer shape of this icon?

I don't know.

**Comment by locness3** [comment]
badge-check seems fitting to me.

**Pull request merged**